sematic/api/endpoints/external_resources.py

```diff
@@ -79,7 +82,9 @@ def activate_resource_endpoint(
             "No such resource: {}".format(resource_id), HTTPStatus.NOT_FOUND
         )
     try:
-        activated = record.resource.activate(is_local=False)
+        resource: AbstractExternalResource = record.resource
+        activated = resource.activate(is_local=False)
```

Comment:
If we only intend this for usage in EE features, we shouldn't have it happen for all `ExternalResources`; there is currently one non-EE `ExternalResource` (`TimedMessage`), and likely will be others. We can filter to ones whose module contains `sematic.ee`

Reply:
This code does not know whether `resource` is EE or not. the call to `track_usage` is a no-op for plug-ins who do not inherit from `TrackedPluginMixin`.